src/lib/core/components/Form/types/array.ts

```diff
@@ -16,13 +16,30 @@ import {
     ValidatorsMap,
 } from './';
 
-export type ArrayInputProps = InputProps<FieldArrayValue, ArraySpec>;
-export type ArrayIndependentInputProps = IndependentInputProps<FieldArrayValue, ArraySpec>;
-export type ArrayLayoutProps = LayoutProps<FieldArrayValue, ArraySpec>;
+export type ArrayInputProps<IProps extends Record<string, any> = {}> = InputProps<
```

Comment:
I think it would be better to take the same name for generic type as in the spec `InputProps`

Reply:
we have a type `InputProps` here, because of it i rename generic